client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -204,7 +203,7 @@ const mapStateToProps = ( state ) => {
 	const isEligible = isEligibleForAutomatedTransfer( state, siteId );
 	const isEligibleForHostingTrial =
 		isUserEligibleForFreeHostingTrial( state ) && site && site.plan?.is_free;
-	config.isEnabled( 'hosting-trial' );
```

Comment:
@paulopmt1 we need to revert this file to trunk. We will make these plugins related changes in another PR

Reply:
Here's the PR https://github.com/Automattic/wp-calypso/pull/86968